BuildCompat.py

```diff
@@ -16,30 +15,41 @@
 
 PUBLICIZER = os.environ.get("PUBLICIZER", "./AssemblyPublicizer")
 DOWNLOAD_LIBS = os.environ.get("DOWNLOAD_LIBS", "--download-libs")
-tasks = []
-
-def system(*cmd):
-    sp = Popen(cmd)
-    if not parallel:
-        sp.wait()
-        ec = sp.poll()
-        if ec:
-            raise SystemExit(ec)
-    else:
-        tasks.append(sp)
-
-with open("Source/CombatExtended.sln") as f:
-    for line in f.readlines():
-        line = line.strip()
+
+async def run_command(*cmd):
+    proc = await asyncio.create_subprocess_exec(*cmd)
+    await proc.wait()
+    if proc.returncode != 0:
+        raise SystemExit(proc.returncode)
```

Comment:
This exits non-zero even in parallel build mode.  That is probably a good thing as it would let us use parallel mode in the github action and not incorrectly ship a broken build.  However it leaves the other tasks hanging.  They then spew their standard out and standard error messages after the main script terminates.  

The return values from `run_command` are accumulated by `asyncio.gather`, so `run_command` should simply return the failure code, and its caller should check that error code and handle it.  This *could* be done via try/except, assuming `asyncio` can be told to wait on all even after the first exception is thrown, but that does not appear to be the default from `asyncio.gather`.  Printing a summary (passed / total) at the end is probably the correct solution.